File: Python/Product.py
class Product:

    def __init__(self):
        self.idProduct = "blank"
        self.price = 0

    
    def setIdProduct(self, idProduct):
        self.idProduct = idProduct

    def getIdProduct(self):
        return self.idProduct

    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price

File: PHP/Product.php
<?php

    class Product
    {
        public $idProduct;
        public $price;

        public function __construct($idProduct = "", $price = 0)
        {
            $this->idProduct = $idProduct;
            $this->price = $price;
        }

        public function setIdProduct($idProduct)
        {
            $this->idProduct = $idProduct;
        }
        public function getIdProduct()
        {
            return $this->idProduct;
        }


        public function setPrice($price)
        {
            $this->price = $price;
        }
        public function getPrice()
        {
            return $this->price;
        }

        public function __destruct()
        {
            
        }
    }

?>